src/Main.java

```diff
@@ -1,5 +1,177 @@
 public class Main {
+    static int counter = 1;
+    static Employee[] employeeArray = new Employee[10];
+
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        System.out.println("Курсач");
+
+        String employeeName1 = "Борис ";
+        String employeeName2 = "Олеся ";
+        String employeeName3 = "Виталий ";
+        String employeeName4 = "Наталья ";
+        String employeeName5 = "Андрей ";
+
+        String employeeName6 = "Василий ";
+        String employeeName7 = "Анжела ";
+        String employeeName8 = "Ольга ";
+        String employeeName9 = "Семен ";
+        String employeeName10 = "Геннадий ";
+
+        String employeeSecondName1 = "Сергеевич ";
+        String employeeSecondName2 = "Владимировна ";
+        String employeeSecondName3 = "Витальевич ";
+        String employeeSecondName4 = "Андреевна ";
+        String employeeSecondName5 = "Васильевич ";
+
+        String employeeSecondName6 = "Степанович ";
+        String employeeSecondName7 = "Петровна ";
+        String employeeSecondName8 = "Викторовна ";
+        String employeeSecondName9 = "Семенович ";
+        String employeeSecondName10 = "Петрович ";
+
+        String employeeLastName1 = "Иванов";
+        String employeeLastName2 = "Степанова";
+        String employeeLastName3 = "Евсеев";
+        String employeeLastName4 = "Кузьмина";
+        String employeeLastName5 = "Новожилов";
+
+        String employeeLastName6 = "Мальцев";
+        String employeeLastName7 = "Сергеева";
+        String employeeLastName8 = "Шубина";
+        String employeeLastName9 = "Турбина";
+        String employeeLastName10 = "Малахов";
+
+        int deptName1 = 1;
+        int deptName2 = 2;
+        int deptName3 = 3;
+        int deptName4 = 4;
+        int deptName5 = 5;
+
+        int deptName6 = 1;
+        int deptName7 = 2;
+        int deptName8 = 3;
+        int deptName9 = 4;
+        int deptName10 = 5;
+
+        int employeeSalary1 = 85000;
+        int employeeSalary2 = 75000;
+        int employeeSalary3 = 65000;
+        int employeeSalary4 = 95000;
+        int employeeSalary5 = 83000;
+
+        int employeeSalary6 = 87000;
+        int employeeSalary7 = 89000;
+        int employeeSalary8 = 54000;
+        int employeeSalary9 = 85500;
+        int employeeSalary10 = 10000;
+
+        employeeArray[0] = new Employee(employeeName1, employeeSecondName1, employeeLastName1, deptName1, employeeSalary1);
```

Comment:
Много лишнего кода. Создать сотрудника можно проще, не плодя лишних переменных
```suggestion
        employeeArray[0] = new Employee("Борс", "Сергеевич", "Иванов", 1, 85000);
```
Тогда лишние переменные выше можно будет удалить (они все равно больше нигде не нужны